pytket/pytket/qasm/qasm.py

```diff
@@ -1039,6 +1039,45 @@ def circuit_from_qasm_wasm(
     return circuit_from_qasm(input_file, encoding=encoding, maxwidth=maxwidth)
 
 
+def circuit_from_qasm_str_wasm(
+    qasm_str: str,
+    wasm: bytes,
+    maxwidth: int = 32,
+) -> Circuit:
+    """A method to generate a tket Circuit from a qasm string and external WASM module.
+
+    :param qasm_str: wasm string
+    :param wasm: base64 encoded wasm file as a string
```

Comment:
I think this is incorrect, it's not expecting the base64 encoding but the raw wasm binary (as `bytes`, not `str`). At least that is what it looks like the test is giving it.

Reply:
done see [8baa0a2](https://github.com/CQCL/tket/pull/1898/commits/8baa0a27d34b012bd5b2d9322dde551ed5f451c4)